subnet_test.go

```diff
@@ -86,7 +86,6 @@ var subnetResponse = `
         "space": "space-0",
         "id": 34,
         "resource_uri": "/MAAS/api/2.0/subnets/34/",
-        "dns_servers": [],
```

Comment:
How about explicitly using `"dns_server": null` above instead? Can it be null coming from MAAS or it's just omitted?